docs/tables/crowdstrike_zta_assessment.md

```diff
@@ -0,0 +1,46 @@
+# Table: crowdstrike_zta_assessment
+
+Zero Trust Assessment is a Falcon Insight feature that monitors the native OS security settings and applied sensor policies of Windows 10 and macOS endpoints to gauge the device security posture of managed assets within an organization. The metrics derived from these assessments can be used by IdP, NAC, and other solutions to provide additional data points around device health and security posture as part of their respective conditional access capabilities.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  device_id,
+  aid,
+  assessment,
+  event_platform
+from
+  crowdstrike_zta_assessment
+```
+
+### List ZeroTrust assessments with assessment score over a threshold
+
+```sql
+select
+  device_id,
+  aid,
+  assessment,
+  event_platform,
+  assessment ->> 'overall' as overall
+from
+  crowdstrike_zta_assessment
+where
+  (assessment ->> 'overall')::int > 92
+```
+
+### List device_ids with firewalls disabled
```

Comment:
```suggestion
### List device IDs with firewalls disabled
```

We usually use the friendly name instead of the column name (unless it's confusing without the column name) in example descriptions